aws/network-primitives/scripts/create_bootstrap_node_compose_file.sh

```diff
@@ -63,7 +64,7 @@ services:
       - "--keypair"
       - \${DSN_NODE_KEY}
       - "--listen-on"
-      - /ip4/0.0.0.0/tcp/30533
+      - /ip4/0.0.0.0/udp/30533/quic-v1
```

Comment:
As mentioned in Slack, this should listen on both UDP and TCP, you just replaced TCP with UDP